README.md

```diff
@@ -1,21 +1,36 @@
 # greensenti
 
 ![CI](https://github.com/KhaosResearch/greensenti/actions/workflows/ci.yml/badge.svg)
-<img alt="Code style: black" src="https://img.shields.io/badge/code%20style-black-000000.svg"></a>
+![Release](https://github.com/KhaosResearch/greensenti/actions/workflows/release.yml/badge.svg)
+![GitHub Release](https://img.shields.io/github/release/KhaosResearch/greensenti.svg)
+![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)
 
 ## Setup
 
-From PyPI:
+### Pip
+
+Use the following command to install from the latest release on PyPI:
 
 ```console
-$ pip install greensenti
-$ pip install greensenti[gcloud] # For support to download from Google Cloud storage
+python -m pip install "greensenti[complete]"
 ```
 
-From source code:
+or install only the core package with:
```

Comment:
Doesn't it make more sense to say how to install the core first, and then offer the user the option of the extra dependencies?

Reply:
l'll make that change right away.